notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -243,26 +243,35 @@
        "<table border=\"1\" class=\"dataframe\">\n",
```

Comment:
<p>test comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>reply to c1</p>